monster.py

```diff
@@ -0,0 +1,51 @@
+import random
+
+
+class Monster:
+    proportion = 0.90
+
+    def __init__(self, name_of_monster, hero_level):
+        self._name = name_of_monster
+        self._level = random.randint(hero_level - 1, hero_level + 1)
+        self._hp = int(self._level * self.proportion + 1)
+        self._damage = int(self._level * self.proportion + 1)
+
+    def attack(self, hero):
+        hero.reduce_health(self)
+
+    def reduce_health(self, hero):
+        self.hp -= hero.damage
+        if self.hp - hero.damage < 0:
```

Comment:
this should be done before the update of the hp, or just check if the hp is less then 0